api/consensus.go

```diff
@@ -45,3 +41,29 @@ func (srv *Server) consensusHandler(w http.ResponseWriter, req *http.Request) {
 	}
 	writeError(w, "unrecognized method when calling /consensus", http.StatusBadRequest)
 }
+
+// consensusBlockHandlerGET handles a GET request to /consensus/block.
+func (srv *Server) consensusBlockHandlerGET(w http.ResponseWriter, req *http.Request) {
+	height, err := scanBlockHeight(req.FormValue("height"))
+	if err != nil {
+		writeError(w, err.Error(), http.StatusBadRequest)
+		return
+	}
```

Comment:
You should be able to `fmt.Sscan` directly into a `types.BlockHeight`. Since the type is unsigned, `Sscan` will throw an error if you supply a negative value.